src/patternfly/components/Page/page.scss

```diff
@@ -411,6 +411,12 @@ $pf-c-page--breakpoint-map: build-breakpoint-map("base", "sm", "md", "lg", "xl",
       flex: 1;
       max-width: var(--pf-c-page--section--m-limit-width--MaxWidth);
     }
+
+    &.pf-m-center {
+      > .pf-c-page__main-body {
+        margin: auto;
```

Comment:
Also is there a reason we wouldn't use align-items here since the parent is a flex container? And wdyt about calling the variation `pf-m-align-[dir]` similar to the dropdown/menu alignment? Would probably be more consistent.